src/swap/types.ts

```diff
@@ -67,18 +97,48 @@ export type SwapError = {
   error: string; // The error message
 };
 
+/**
+ * Note: exported as public Type
+ */
+export type SwapParams = GetQuoteParams | GetSwapParams;
+
+export type SwapAPIParams = GetQuoteAPIParams | GetSwapAPIParams;
+
+/**
+ * Note: exported as public Type
+ */
+export interface Transaction {
+  transaction: TransactionData;
```

Comment:
this is the object developers/users should pass into viem's `signTransaction` (https://viem.sh/docs/actions/wallet/signTransaction.html)